packages/core/src/App/Constants/routes-config.js

```diff
@@ -41,7 +41,7 @@ const Page404 = lazy(() => import(/* webpackChunkName: "404" */ 'Modules/Page404
 const Trader = lazy(() => {
     const el_head = document.querySelector('head');
     const el_main_css = document.createElement('link');
-    el_main_css.href = '/css/trader.main.css';
+    el_main_css.href = getUrlBase('/css/trader.main.css');
```

Comment:
@mahdipourziaei-binary can we move the getUrlBase to shared?

Reply:
@Mahboobeh-binary As per discussions, this will be done in another PR.